hack/debug.sh

```diff
@@ -61,14 +62,25 @@ log_system () {
 }
 
 do_master () {
+    # Find the config file
+    config=$(ps wwaux | grep -v grep | sed -ne 's/.*openshift.*--\(master-\)\?config=\([^ ]*\.yaml\).*/\2/p')
```

Comment:
Our normal configuration doesn't use a config file for the master, it just passes stuff on the command line. Also, if the master is also a node, then this expression will match the node process and end up grabbing its config file.